src/disasm_wrapper.c

```diff
@@ -207,11 +207,24 @@ struct intercept_disasm_result
 intercept_disasm_next_instruction(struct intercept_disasm_context *context,
 					const unsigned char *code)
 {
+	static const unsigned char endbr64[] = {0xf3, 0x0f, 0x1e, 0xfa};
```

Comment:
Just to be sure: have you tested this? In particular, whether the byte order is correct (LE vs BE).

Reply:
Yes, originally I wrote "endbr64" in the assembly tests. Also, this instruction is used all over glibc in newer version (every branch destination has to arrive at an endbr64 instruction).

Then I had to change the assembly test to not use the instruction name, because the older assembler on docker images doesn't know about this instruction yet.

Reply:
In fact now I just added the `endbr64` mnemonic to the assembly tests.
in `test/CMakelists.txt`, this assembly test is only included if the assembler supports the instruction.
I also added builds with newer Ubuntu, so this test is only executed in those travis builds.